Cargo.toml

```diff
@@ -1,6 +1,30 @@
+[package]
+name = "tower"
+version = "0.1.0"
+authors = ["Carl Lerche <me@carllerche.com>"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/tower-rs/tower"
+description = """
+Tower is a library of modular and reusable components for building robust
+clients and servers.
+"""
+categories = ["asynchronous", "network-programming"]
+keywords = ["io", "async", "non-blocking", "futures", "service"]
```

Comment:
I'd probably put `publish = false` on a newline, just to protect ourselves.

Reply:
done!